dCommon/GeneralUtils.cpp

```diff
@@ -57,38 +57,21 @@ std::u16string GeneralUtils::ASCIIToUTF16(const std::string& string, size_t size
     return ret;
 }
 
-//! Converts a (potentially-ill-formed) UTF-16 string to UTF-8
-//! See: <http://simonsapin.github.io/wtf-8/#decoding-ill-formed-utf-16>
+//! Converts an UCS-2 / UTF-16 to std::string (ASCII)
 std::string GeneralUtils::UTF16ToWTF8(const std::u16string& string, size_t size) {
```

Comment:
```suggestion
std::string GeneralUtils::UTF16ToUTF8(const std::u16string& string, size_t size) {
```

Again, if you change the implementation, change the name 😉